### ISSUE TITLE
Support multiple tags/ids per sector/line in UDMF maps

### ISSUE THREAD

**Issue by eevee** [sirjuddington/SLADE#328]
It's not in a stable release yet, but ZDoom now supports multiple tags/ids on a sector or linedef by jamming the rest of them into the `moreids` property as a comma-delimited string.  Ideally, SLADE would support this transparently, treating both properties as a single list.

_No_ idea how the UI should work, or how this affects the "new tag" button.

**Comment by Monsterovich** [comment]
>treating both properties as a single list.

Please, don't make tag list complicated. If we have two UDMF variables, we should make them separated.

============  -------------------
=         32       =  - NEW TAG  -
============  -------------------

============
=   33,34,35   =  
============

**Comment by eevee** [comment]
I don't know that inheriting UDMF's backwards-compatibility concerns is a good idea.  All the internals and UI would need to understand multiple ids, either way.  Separate fields just makes that more complicated.

**Comment by Monsterovich** [comment]
I got an idea. We can make a table for that. The first row will be regular "tag" value, other rows will be parsed as "moreids" value. In this table "new tag" button must work by the same way, but just for currently selected row.

**Comment by eevee** [comment]
Hm, how would you add a new row?

It seems like the "new tag" button is most useful when you don't already have a tag, so maybe it should always add a new one?  You can just delete the old one if that's what you want.

**Comment by Monsterovich** [comment]
>It seems like the "new tag" button is most useful when you don't already have a tag, so maybe it should always add a new one? You can just delete the old one if that's what you want.

Yeah, but if you already have a tag, it should generate a new number for selected tag in a row.

**Comment by eevee** [comment]
Other concerns:

* How is this exposed in the prop grid?
* How does this affect "change tag" mode?

**Comment by eevee** [comment]
This is now in a stable release, ZDoom 2.8.0.

Also note that this is a ZDoom-specific property, not part of the UDMF base spec, so it shouldn't be exposed in other ports.

**Comment by jmfergeau** [comment]
Please add this. I can't use UDB on linux because it's too unstable and i'm doing everything on slade. I don't want to use UDB just for that :(

**Comment by sirjuddington** [comment]
This is implemented in 3.3.0 beta 1
